app/serializers/disability_claim_base_serializer.rb

```diff
@@ -65,4 +69,21 @@ def date_from_string(*keys, format: '%m/%d/%Y')
       Date.strptime(s, format)
     end
   end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def phase_from_keys(*keys)
+    case s = object.data.dig(*keys)&.downcase
+    when 'claim received' then 1
```

Comment:
Could use a hash constant here, might allow you to remove the rubocop disable here
Something like
```ruby
PHASE_MAPPING = {
'claim_received' => 1, 
...
}

def phase_from_keys(*keys)
  s = object.data.dig(*keys)&.downcase
  phase = PHASE_MAPPING[s]
  Rails.logger.error "Expected EVSS #{keys} to be a phase. Got '#{s}'." unless phase
  phase
end
```